examples/grafana/httproute_filters/README.md

```diff
@@ -0,0 +1,41 @@
+# HTTPRoute with Request/Response Filters Example
```

Comment:
The main goal of the examples we have is to offer some basic manifests, not to cover all possible scenarios. Adding docs on custom filters seems to be out of scope, end users can explore that in the Gateway API docs.
Let's see what other maintainers think of this.

Reply:
I think the files with this example are still there.

Reply:
Removed httproute_filters example entirely. Only httproute_basic remains.

Reply:
`httproute_basic` is not the only example that is left, there is also `httproute_tls`.